packages/job-worker/src/db/collections.ts

```diff
@@ -58,6 +60,25 @@ export interface ICollection<TDoc extends { _id: ProtectedString<any> }> {
 	replace(doc: TDoc | ReadonlyDeep<TDoc>): Promise<boolean>
 
 	bulkWrite(ops: Array<AnyBulkWriteOperation<TDoc>>): Promise<unknown>
+
+	/**
+	 * Watch the collection for changes
+	 * Note: Not available in all workers/situations
```

Comment:
This is a quite scary comment :) could you elaborate?

Reply:
better?